uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/callbacks/DefaultErrorCallback.java

```diff
@@ -23,7 +23,9 @@
 
 /**
  * Default Error handler for all Portable Exceptions
+ * @deprecated Please use GenericErrorPopup instead.
  */
+@Deprecated
```

Comment:
Just a question related to this deprecation: does it mean that there are still places in the workbench, that will display unexpected errors in "the old way"? Or will these be handled universally using the new GenericErrorPopup?

Reply:
Yes. This class is still used, but only by code actively wanting to do a custom error handling. I didn't dig in detail why, but the default error handling has changed, so not much to worry here.